cmd/metrics-v3-cluster-ilm.go

```diff
@@ -0,0 +1,53 @@
+// Copyright (c) 2024 MinIO, Inc.
+//
+// # This file is part of MinIO Object Storage stack
+//
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU Affero General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU Affero General Public License for more details.
+//
+// You should have received a copy of the GNU Affero General Public License
+// along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+package cmd
+
+import (
+	"context"
+)
+
+const (
+	expiryPendingTasks             = "expiry_pending_tasks"
+	transitionActiveTasks          = "transition_active_tasks"
+	transitionMissedImmediateTasks = "transition_missed_immediate_tasks"
+	transitionPendingTasks         = "transition_pending_tasks"
+	versionsScanned                = "versions_scanned"
+)
+
+var (
+	ilmExpiryPendingTasksMD             = NewCounterMD(expiryPendingTasks, "Number of pending ILM expiry tasks in the queue")
+	ilmTransitionActiveTasksMD          = NewCounterMD(transitionActiveTasks, "Number of active ILM transition tasks")
+	ilmTransitionMissedImmediateTasksMD = NewCounterMD(transitionMissedImmediateTasks, "Number of missed immediate ILM transition tasks")
+	ilmTransitionPendingTasksMD         = NewCounterMD(transitionPendingTasks, "Number of pending ILM transition tasks in the queue")
+	ilmVersionsScannedMD                = NewCounterMD(versionsScanned, "Total number of object versions scanned since server start")
```

Comment:
```suggestion
	ilmVersionsScannedMD                = NewCounterMD(versionsScanned, "Total number of object versions checked for ilm actions since server start")
```

The description needs to be corrected here as well.